tests/test_idutils.py

```diff
@@ -39,22 +39,19 @@
     ('http://dx.doi.org/10.1016/j.epsl.2011.11.037', ['doi', 'url', ],
         '10.1016/j.epsl.2011.11.037',
         'http://dx.doi.org/10.1016/j.epsl.2011.11.037'),
-    ('9783468111242', ['isbn', 'ean13'], '', ''),
     ('4006381333931', ['ean13'], '', ''),
     ('73513537', ['ean8'], '', ''),
-    ('1562-6865', ['issn'], '', ''),
+    ('15626865', ['issn', 'pmid'], '1562-6865', ''),
```

Comment:
Hm, not sure how to deal with this. I think we likely have to remove PubMed ID checking since an integer is simply to generic. Anyway, shouldn't stop this from being integrated.